.github/workflows/ci-build.yml

```diff
@@ -38,63 +38,26 @@ jobs:
 
   cargo-c:
 
-    env:
-      RUST_BACKTRACE: 1
-      RUSTC_WRAPPER: sccache
-      SCCACHE_CACHE_SIZE: 300M
-      SCCACHE_DIR: /home/runner/.cache/sccache
-      SCCACHE_IDLE_TIMEOUT: 0
-
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v2
 
-    - name: Install sccache
-      env:
-        LINK: https://github.com/mozilla/sccache/releases/download
-        SCCACHE_VERSION: 0.2.13
-      run: |
-        SCCACHE_FILE=sccache-$SCCACHE_VERSION-x86_64-unknown-linux-musl
-        mkdir -p $HOME/.local/bin
-        curl -L "$LINK/$SCCACHE_VERSION/$SCCACHE_FILE.tar.gz" | tar xz
-        mv -f $SCCACHE_FILE/sccache $HOME/.local/bin/sccache
-        echo "::add-path::$HOME/.local/bin"
-
     - name: Install stable
       uses: actions-rs/toolchain@v1
       with:
         profile: minimal
         toolchain: stable
         override: true
 
-    - name: Cache cargo registry
-      uses: actions/cache@v1
-      with:
-        path: ~/.cargo/registry/cache
-        key: cargo-registry-${{ hashFiles('**/Cargo.lock') }}
-        restore-keys: |
-          cargo-registry-
-
-    - name: Cache sccache output
-      uses: actions/cache@v1
-      with:
-        path: ~/.cache/sccache
-        key: sccache-${{ hashFiles('**/Cargo.*') }}
-        restore-keys: |
-          sccache-
-
-    - name: Start sccache server
-      run: |
-        sccache --start-server
-
     - name: Install cargo-c
       run: |
         cargo install cargo-c
 
     - name: Run cargo-c
       run: |
-        cargo cinstall --manifest-path=av_metrics/Cargo.toml \
+        cargo cinstall --all-features \
```

Comment:
I wouldn't use `all-features` though.

Reply:
I've used `all-features` to fix this warning:
`warning: package replacement is not used: https://github.com/rust-lang/crates.io-index#av-metrics:0.5.0`